src/assets/SSSAsset.js

```diff
@@ -0,0 +1,35 @@
+const postcss = require('postcss');
+const sugarss = require('sugarss');
+const CSSAsset = require('./CSSAsset.js');
+
+class SSSAsset extends CSSAsset {
+  constructor(name, options) {
+    options.css_parser = "sugarss";
+    super(name, options);
+    this.type = 'css';
+  }
+
+  parse(code) {
+    let root = postcss.parse(code, {from: this.name, to: this.name, syntax: sugarss});
```

Comment:
`syntax` => `parser`.

We need to set SugarSS only as input language (parser) and keep output language (stringifier) as default CSS

Reply:
I think I also tried it initially, but for some reason postcss still parsed code as if it was CSS

Reply:
I will try to run tests after Def Con

Reply:
I can try if you want.

Reply:
Yeap, it should be `parser:`.

Reply:
So all that remains is to automate the configuration?